src/updateDisplay.js

```diff
@@ -0,0 +1,58 @@
+import addToStorage from './addToStogage.js';
+import updateIndices from './updateIndices.js';
+
+const updateDisplay = () => {
+  const data = JSON.parse(localStorage.getItem('data')) || [];
+  const listTodo = document.querySelector('.to-do-list');
+  listTodo.innerHTML = '';
+
+  data.forEach((todo, index) => {
+    const listItem = document.createElement('li');
+    const container = document.createElement('div');
+    container.className = 'textContainer';
+    const iconDiv = document.createElement('div');
+    iconDiv.innerHTML = '<i class="fa-solid fa-trash"></i>';
+    iconDiv.addEventListener('click', (index) => {
+      data.splice(index, 1);
+      localStorage.setItem('data', JSON.stringify(data));
+      updateDisplay();
+      updateIndices();
+    });
```

Comment:
- [OPTIONAL] I noticed that the delete function for a single button doesn't delete properly. In the demonstration below, I have three tasks: **_Play football_**, **Code**, **_Watch movie_**. When I try to delete the **_Watch Movie_** tasks, it deletes the **_Play Football_** task. Kindly fix this error as it would not be a good user experience in real life. 👍 😎 

![chrome-capture-2023-7-25](https://github.com/enocol/webpack/assets/77106962/8e7dd4c1-b684-437c-b564-cab9307219aa)